### DISCUSSION TITLE
Multi-select support for "drag & drop" and "field updates"

### DISCUSSION THREAD

**Discussion by visormatt** [community/community#5516] [Projects and Issues] [answered]
# Overview

As I look to migrate some existing projects into this awesome new Beta version it appears as though you can't actually move multiple issues (although selected) across columns in a board. This general idea would/could also apply to applying tags or updating status to multiple issues as once. Love to get any tips, tricks or thoughts! 

**Keep up the great work Github!**

**Comment by rileybroughten** [comment, marked as answer]
Bulk drag and drop on boards is now supported - check out the [changelog announcement](https://github.blog/changelog/2023-05-25-github-issues-projects-may-25th-update/#%F0%9F%92%AA-%E2%8C%A8%EF%B8%8F-bulk-updates-and-keyboard-navigation-on-boards) for more details!
